doc/reference/native-tokens/02-getting-started.md

```diff
@@ -0,0 +1,582 @@
+## Getting started with native tokens
+
+### Prerequisites
+
+This section is for advanced users who are proficient with the Cardano-node [command line interface (CLI)](https://github.com/input-output-hk/cardano-node/blob/master/doc/reference/cardano-node-cli-reference.md).
+
+This section describes how to:
+
+- create new currencies and assets;
+- submit and send transactions containing multi-asset tokens;
+- send and receive token bundles;
+- manage your addresses and values.
+
+> Note: Users who do not need to create new assets (“token holders”) will be able to send and receive existing multi-asset tokens using a wallet such as Daedalus or Yoroi, and with no requirement to use any CLI commands.
+
+To start, please ensure that you are familiar with setting up and operating the [Cardano node](https://github.com/input-output-hk/cardano-node). Alternatively, see instructions on how to [start your node](https://github.com/input-output-hk/cardano-node/blob/master/doc/stake-pool-operations/2_start_your_nodes.md) to submit the commands. You will not need to set up and start a full block producing node (“stake pool”), just a much simpler relay node. This node will need to connect to a Cardano network that is capable of processing native tokens (e.g., the native token pre-production environment (PPE), or the Cardano Mainnet once it has been upgraded).
+
+### Configuring cardano-node to run using cardano-cli
```

Comment:
A general version of this section should live in the `README.md` in the `cardano-node` dir and then referenced here.

Reply:
This is pushed. Closing now